src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -78,35 +93,27 @@ private static void removeHandlers(Logger logger) {
                 .forEach(logger::removeHandler);
     }
 
-    /**
-     * Adds the {@code fileHandler} to the {@code logger}. <br>
-     * Creates {@code fileHandler} if it is null.
-     */
-    private static void addFileHandler(Logger logger) {
-        try {
-            if (fileHandler == null) {
-                fileHandler = createFileHandler();
-            }
-            logger.addHandler(fileHandler);
-        } catch (IOException e) {
-            logger.warning("Error adding file handler for logger.");
-        }
-    }
-
     /**
      * Creates a {@code FileHandler} for the log file.
+     *
+     * @param level the logging level of the file handler
      * @throws IOException if there are problems opening the file.
      */
-    private static FileHandler createFileHandler() throws IOException {
+    private static FileHandler createFileHandler(Level level) throws IOException {
         FileHandler fileHandler = new FileHandler(LOG_FILE, MAX_FILE_SIZE_IN_BYTES, MAX_FILE_COUNT, true);
         fileHandler.setFormatter(new SimpleFormatter());
-        fileHandler.setLevel(currentLogLevel);
+        fileHandler.setLevel(level);
         return fileHandler;
     }
 
-    private static ConsoleHandler createConsoleHandler() {
+    /**
+     * Creates a {@code ConsoleHandler} for the log file.
+     *
+     * @param level the logging level of the console handler
+     */
+    private static ConsoleHandler createConsoleHandler(Level level) {
         ConsoleHandler consoleHandler = new ConsoleHandler();
-        consoleHandler.setLevel(currentLogLevel);
+        consoleHandler.setLevel(level);
         return consoleHandler;
     }
```

Comment:
I'm wondering if these two methods can be in-lined. Although not strictly SLAP, it might reduce the amount of code without reducing readability. What do you think?

Reply:
Since these methods are only referred once and are individually not very long, I think they can be inlined since we did not include any plan to create more custom loggers.